apache-php/docker-compose.yaml

```diff
@@ -0,0 +1,9 @@
+version: "3.7"
+services:
+  web:
+    build: app
+    ports: 
+      - '8008:80'
+    volumes:
+      - ./app:/var/www/html/
+      
```

Comment:
nit: add a new line at  the end of the file

Reply:
done